libs/web/project.json

```diff
@@ -3,5 +3,20 @@
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "libs/web",
   "projectType": "library",
-  "tags": []
+  "tags": [],
```

Comment:
just making a record that this is to exclude the integration tests from CI cause they take too long